bin/git-guilt

```diff
@@ -49,19 +39,29 @@ then
 else
     PATTERN='s/^author //p'
 fi
-sed -n "$PATTERN" $SINCE_LOG | sed 's/^\(.\)/- \1/' >> $MERGED_LOG
-sed -n "$PATTERN" $UNTIL_LOG | sed 's/^\(.\)/+ \1/' >> $MERGED_LOG
+
+for file in $(git diff --name-only "$@")
+do
+    test -n "$DEBUG" && echo "git blame $file"
+    # $1 - since $2 - until
+   
```

Comment:
Either you are missing a space character or the line should be empty overall. :wink: